src/views/homepage/components/featured-content-grid/index.tsx

```diff
@@ -0,0 +1,51 @@
+import Card from 'components/card'
+import Heading from 'components/heading'
+import Text from 'components/text'
+import s from './featured-content-grid.module.css'
+
+const FeaturedContentGrid = () => {
+	return (
+		<div className={s.root}>
+			<div className={s.gridAreaA}>
+				<Card className={s.searchCard} elevation="base">
+					<div className={s.searchCardText}>
+						<Heading level={2} size={400} weight="bold">
+							Search with ease
+						</Heading>
+						<Text size={200} weight="medium">
+							Explore highly regarded resources
+							<br />
+							on a wide range of topics.
```

Comment:
❓ Does the education team want the ability to edit this content?

Reply:
As far as I know, this page won't be authorable in its initial phases.